.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "static/JSON-js"]
```

Comment:
(see below: can you remove this as and when the debugging is over; I don't mind submodules in general but if we can avoid any that'd be better :))